### PR TITLE
ThirtyBees App installer

### PR DESCRIPTION
**Pull request by aitorroma**
### What is ThirtyBees?

ThirtyBees is an open-source e-commerce platform derived from PrestaShop 1.6. It is designed to provide a lighter, faster, and more stable solution for creating and managing online stores. Like PrestaShop, ThirtyBees is free and allows users to customize and efficiently run their e-commerce businesses.

**Key Features of ThirtyBees:**
- **Improved Stability and Performance:** ThirtyBees incorporates significant enhancements compared to its predecessor, focusing on seamless functionality.
- **Compatibility with PrestaShop 1.6 Modules and Themes:** Most modules and themes built for PrestaShop 1.6 are compatible, making migration straightforward.
- **Responsive Design:** Built with Bootstrap, it ensures optimal display across mobile and desktop devices.
- **One-Page Checkout:** Offers a streamlined checkout process with options for a single-page or multi-step checkout flow to enhance user experience.
- **SEO Optimization Tools:** Includes built-in tools to help improve search engine rankings.
- **Multi-store and Multi-currency Support:** Enables the management of multiple stores and supports various currencies, including cryptocurrencies.
- **Advanced Tax and Discount Management:** Features comprehensive support for international tax systems and advanced tools for configuring discounts and coupons.

**Why Choose ThirtyBees?**
ThirtyBees stands out as a robust and flexible alternative for merchants seeking a high-performance e-commerce platform. Its focus on stability, user-friendly features, and compatibility with existing PrestaShop modules makes it an excellent choice for businesses aiming to scale and optimize their online operations.

### CODE REVIEW

File: web/src/app/WebApp/Installers/ThirtyBees/ThirtyBeesSetup.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                "--lastname=" . $options["thirtybees_account_last_name"],
+                "--password=" . $options["thirtybees_account_password"],
+                "--email=" . $options["thirtybees_account_email"],
+                "--domain=" . $this->domain,
+                "--ssl=" . $ssl_enabled,
+            ],
+            $status,
+        );
+
+    // Eliminar el directorio de instalación
```
Comment (pokoli) [inline web/src/app/WebApp/Installers/ThirtyBees/ThirtyBeesSetup.php]: Better to write it in English: 
```suggestion
    // Delete install directory
```

Comment (aitorroma) [reply web/src/app/WebApp/Installers/ThirtyBees/ThirtyBeesSetup.php]: Thanks @pokoli 

Sabes como arreglar:  Lint / Prettier (pull_request) Failing after 32s

Comment (pokoli) [reply web/src/app/WebApp/Installers/ThirtyBees/ThirtyBeesSetup.php]: I think you should just run: `npx prettier --write ThrityBeesSetup.php` And then `npm prettier --check .` which is the command run by the CI to check everything is in place.

**Review by jaapmarcus — APPROVED** [review]

File: web/src/app/WebApp/Installers/ThirtyBees/ThirtyBeesSetup.php (hunk 1 above)
Comment (jaapmarcus) [reply web/src/app/WebApp/Installers/ThirtyBees/ThirtyBeesSetup.php]: Ran it myself... and committed the changes

**Pull request merged**